### DISCUSSION TITLE
Discord Bot Customization

### DISCUSSION THREAD

**Discussion by gnarlyvr** [gameserverapp/Platform#564] [Suggestions]
Allow direct editing of the embeds for discord bot server status, chat, etc....

This would allow server owners to customize their bot's messages and add a personal touch.

Currently my status bot posts the server ip and direct connect link, which do not work with Ark Survival Ascended, so it is taking up space and causing confusion for players.

**Comment by PR3SIDENT** [comment]
This would be awesome to see.

**Comment by mvaessen** [comment]
Thanks for the suggestions. Most of these items have been added, or overlap with other existing suggestions.
